corehq/apps/style/templates/style/base.html

```diff
@@ -329,33 +344,92 @@
 
         {# javascript below this line #}
 
+        {% if request.use_requirejs %}
+            <script src="{% static 'hqwebapp/js/lib/require.js' %}"></script>
+            <script type="text/javascript">
+                requirejs.config({
+                    baseUrl: "{% block requirejs_url %}{% endblock %}",
```

Comment:
Alternatively, we could not set a `baseUrl` and instead just add all scripts to `paths`, as was done in https://github.com/dimagi/commcare-hq/pull/12359/commits/3ac972316148a05e79fb26f8ab71f5d5e8c8268f